packages/ui/src/components/GalleryPost/GalleryPost.tsx

```diff
@@ -68,25 +55,32 @@ export default function GalleryPost({
     [post]
   );
 
+  const textPostIsJustReference = useMemo(
```

Comment:
Can we extract these helpers + combine with the ones in `GalleryDetailView`? Could be contained in a `getPostMeta` hook or something like that.